pkg/sfu/downtrack.go

```diff
@@ -624,7 +624,7 @@ func (d *DownTrack) WriteRTP(extPkt *buffer.ExtPacket, layer int32) error {
 	d.bytesSent.Add(uint32(hdr.MarshalSize() + len(payload)))
 
 	if tp.isSwitchingToMaxSpatial && d.onMaxSubscribedLayerChanged != nil && d.kind == webrtc.RTPCodecTypeVideo {
-		d.onMaxSubscribedLayerChanged(d, layer)
+		d.onMaxSubscribedLayerChanged(d, d.forwarder.MaxLayer().Spatial)
```

Comment:
the buffer layer is not correct for svc codecs, use forwarder.MaxLayer here, is it ok for other codecs? @boks1971

Reply:
I just hit `Approve` @cnderrauber . I think it should be fine, but let me check this.

Reply:
Yeah, that is going to be incorrect because we overshoot @cnderrauber . I think the best thing would be to include a `MaxSpatialLayer` field in the selector/forwarder return value and use that. Let me make a PR for that real quick. You can try that and if that works, we can skip this PR.

Reply:
Oops, sorry, this is already merged. Anyhow, will make another PR.